src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/activities/ActivitiesMigrationServiceTest.kt

```diff
@@ -545,8 +478,8 @@ class ActivitiesMigrationServiceTest {
             assertThat(rate).isEqualTo(140)
           }
           with(it.scheduleRules[0]) {
-            assertThat(startTime).isEqualTo("${today}T08:00")
-            assertThat(endTime).isEqualTo("${today}T11:30")
```

Comment:
This only tests outputs are the same as inputs - and this format was wrong! Now that we actually collect values depending upon the times we need the correct format.